config/default.yml

```diff
@@ -0,0 +1,4 @@
+Minitest/AssertNil:
+  Description: 'Check if your test uses assert_nil instead of assert_equal(nil, something).'
```

Comment:
Please add an URL of `StyleGuide`.

```diff
 Description: 'Check if your test uses `assert_nil` instead of `assert_equal(nil, something)`.
+StyleGuide: 'https://github.com/rubocop-hq/minitest-style-guide#assert-nil
 Enabled: true
```